CMake/Modules/PlatformTriple.cmake

```diff
@@ -0,0 +1,190 @@
+#
+# Copyright (c) 2017-2025 the rbfx project.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+# Function to generate platform triple: <platform>-<architecture>-<runtime>
+# Example triples:
+#   - emscripten-wasm-lib
+#   - windows-x64-dll
+#   - uwp-arm64-dll
+#   - uwp-arm-lib
+#   - linux-x86_64-lib
+#   - android-aarch64-lib
+#
+# Arguments:
+#   OUT_TRIPLE - Output variable name for the triple
+#   MODE - Optional: "HOST" for host platform, "TARGET" (default) for target platform
+function(rbfx_get_platform_triple OUT_TRIPLE)
```

Comment:
I think that "platform tag" is more undestandable than "platform triple". And more flexible -- what if we want to add compiler too?